src/core/or/connection_edge.c

```diff
@@ -597,31 +597,34 @@ connected_cell_format_payload(uint8_t *payload_out,
   return connected_payload_len;
 }
 
-/* DOCDOCDOC */
-static void
-send_ha_proxy_header(const edge_connection_t *edge_conn,
-                     connection_t *conn)
+/* This is an onion service client connection: Export the client circuit ID
+ * according to the HAProxy proxy protocol. */
+STATIC void
+export_hs_client_circuit_id(const edge_connection_t *edge_conn,
+                            connection_t *conn)
```

Comment:
Since this is HAProxy format specific, I liked the mention of "haproxy" in the name.

Reply:
✓ (0ca04c004d)